meta/autoid/autoid.go

```diff
@@ -912,6 +912,132 @@ func (alloc *allocator) alloc4Sequence(tableID int64) (min int64, max int64, rou
 	return newBase, newEnd, round, nil
 }
 
```

Comment:
I think this file needs to add some unit tests to autoid_test.go.

Reply:
It's still working in progress, don't worry..

Reply:
It's still working in progress, don't worry..